r/configure

```diff
@@ -104,6 +117,8 @@ CPPFLAGS=`"${R_HOME}"/bin/R CMD config CPPFLAGS`
 # If libarrow uses the old GLIBCXX ABI, so we have to use it too
 if [ "$ARROW_USE_OLD_CXXABI" ]; then
   PKG_CFLAGS="$PKG_CFLAGS -D_GLIBCXX_USE_CXX11_ABI=0"
+# else
+#   PKG_CFLAGS="$PKG_CFLAGS -D_GLIBCXX_USE_CXX11_ABI=1"
```

Comment:
Should we remove this?

Reply:
I had this in for debugging because I was looking at a system where old ABI was default. I still want to test on more systems, and I'll clean it up if this isn't something we ever want.